hearpreprocess/pipeline.py

```diff
@@ -870,6 +890,55 @@ def run(self):
         self.mark_complete()
 
 
+class FinalizeCorpus(MetadataTask):
```

Comment:
What's the benefit of including this tar step in the pipeline? Won't we almost all the time be running this on like GCP and then using the generated files? Or are you imagining just running this once per tasks to create the tar and then uploading that for eval?

Reply:
Are we going to host the final task tars on hugging face?

Reply:
I don't want to upload with the script. Too easy to accidentally publish secret tasks. Must do it by hand.